sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/SqlDelightReferenceContributor.kt

```diff
@@ -73,20 +78,37 @@ internal class SqlDelightReferenceContributor : PsiReferenceContributor() {
   }
 }
 
-private fun getKotlinTopLevelTypeAliasFqNameIndex(): KotlinTopLevelTypeAliasFqNameIndex {
+private fun getKotlinTopLevelTypeAliasFqNameIndex(): StubIndexKey<String, KtTypeAlias> {
```

Comment:
actually we can drop this altogether and invoke 'KotlinTopLevelTypeAliasFqNameIndex.get() normally, because the workaround was needed for version before AS Flamingo

Reply:
The new version requires using the helper companion object: https://github.com/JetBrains/intellij-community/blob/master/plugins/kotlin/base/indices/src/org/jetbrains/kotlin/idea/stubindex/KotlinTopLevelTypeAliasFqNameIndex.kt

Reply:
at least I assume so because its not longer a static method to call `get` and the type you call it through has an internal constructor

Reply:
Yeah it will income it on companion object, see https://github.com/JetBrains/intellij-community/blob/9e97ea132043bf90424289d0ffd0c223460d6e71/plugins/kotlin/j2k/new/src/org/jetbrains/kotlin/nj2k/JKResolver.kt#L50

Reply:
ah! I see, alright, will update

Reply:
updated